test/integration/pipeline/transfer_asset_inter_domain_test.cpp

```diff
@@ -16,7 +16,9 @@
  */
 
 #include "backend/protobuf/from_old_model.hpp"
+#include "cryptography/crypto_provider/crypto_signer_impl.hpp"
 #include "integration/pipeline/tx_pipeline_integration_test_fixture.hpp"
+#include "backend/protobuf/from_old_model.hpp"
```

Comment:
1. This include has written twice. Another on line 18
2. This `include` looks like not allowed here. Ask a question which task is responsible for solving it?